test/unit/ngsiv2/contextRequests/createBidirectionalDevice.json

```diff
@@ -2,7 +2,7 @@
   "id": "Second MQTT Device",
   "type": "AnMQTTDevice",
   "location": {
-    "type": "geo:point",
-    "value": "0, 0"    
+    "type": "geo:json",
+    "value": {"type":"Point","coordinates":[0,0]}  
```

Comment:
Shouldn't be unit test failing before this fix (I mean, in master)?

Reply:
If you re-start the last run it should now fail. The change has occurred in the downstream iotagent-node-lib 

```json
"iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
```

... is now picking up https://github.com/telefonicaid/iotagent-node-lib/commit/a7eb2605d342  commited 20 days ago, but the last CI test run was 23 days ago.


If you really want to pick these up add a daily chron job to the github action CI:

```yaml
on:
  schedule:
    - cron:  '0 7 * * 1,4'
    # scheduled at 07:00 every Monday and Thursday
  workflow_dispatch:  # click the button on Github repo!
```

Reply:
You are right. Now I see the red "X" at master :)

NTC